client/features/shoppingCart/shoppingCartSlice.js

```diff
@@ -0,0 +1,37 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+
+// Fetches this user's shopping cart
+export const fetchAllUserItemsAsync = createAsyncThunk(
+  'GET AllUserItems',
+  async (userId) => {
+    const { data } = await axios.get(`/api/orders/${userId}`);
```

Comment:
i was planning to do GET /api/users/:id/cart ... where :id is the userId, but this works too - but when it comes to the admin side of the site we may want orders/:id free for specific order queries

Reply:
Ahh ok, that makes sense. I'll move this block to api/users/:id/cart and change this orders/:id back to the placeholder you had.